i18n/en.json

```diff
@@ -2196,7 +2195,7 @@
   "delete_channel.cancel": "Cancel",
   "delete_channel.confirm": "Confirm ARCHIVE Channel",
   "delete_channel.del": "Archive",
-  "delete_channel.question": "This will archive the channel from the team and make its contents inaccessible for all users. \n \nAre you sure you wish to archive the **{display_name}** channel?",
+  "delete_channel.question": "This will archive the channel from the team and remove it from the user interface. A copy of the channel is kept on the server in case it is needed for audit purposes.\n \nAre you sure you wish to archive the **{display_name}** channel?",
```

Comment:
> ... in case it is needed for audit purposes

This assumes a specific use case and I'm not sure if it's always applicable to everyone. It'd be sufficient to just clarify data isn't deleted from server.

Reply:
Good point; I've reworded it. LMKWYT.